pianos.geojson

```diff
@@ -98,6 +98,20 @@
           38.1824001
         ]
       }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "description": "Piano outside business in downtown",
+        "type": "business"
```

Comment:
What does `business` mean? Maybe we will have to add a section about the `type` parameter. 😄

Reply:
I'm not sure what type of business it is, but I knew it wasn't a shop or a restaurant.

Reply:
Ah, ok! 😁

Reply:
Thanks!!!